test/front_end/test_front_end.py

```diff
@@ -238,3 +239,34 @@ def test_import_ato(bob: Bob, tmp_path):
 
     r1 = Bob.get_node_attr(node, "r1")
     assert isinstance(r1, F.Resistor)
+
+
+@pytest.mark.parametrize(
+    "module,count", [("A", 0), ("B", 1), ("C", 2), ("D", 3), ("E", 4)]
+)
+def test_traceback(bob: Bob, module: str, count: int):
```

Comment:
Not the best test, but also cheap as chips. I shhhhoould probably improve it a bit